composer.json

```diff
@@ -28,11 +28,12 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^9.3",
-        "squizlabs/php_codesniffer": "^3.6"
+        "squizlabs/php_codesniffer": "^3.6",
+        "laravel/pint": "^1.2"
```

Comment:
@Meldiron did we agree on which version to use ? We might want to keep it consistent across all the PRs

Reply:
We used `dev-main` in example PR, that's what other contributors are using. Let's use `dev-main` here too for `pint`.

Reply:
Done!